packages/paste-website/src/pages/form-elements/select/index.mdx

```diff
@@ -0,0 +1,945 @@
+      <Select id="sort_do" name="sort_do" onChange="">
+        <Option value="node-js">NodeJS</Option>
+        <Option value="java">Java</Option>
+        <Option value="php">PHP</Option>
+      </Select>
+    </Box>
+  </Dont>
+</DoDont>
+
+<DoDont>
+  <Do
+    center
+    title="Do"
+    body="Use a safe and reversible option as the default selected value, or an option that tells users to choose from the list. Prevent users from unintentionally submitting a default value in case they accidentally skip over a field."
+  >
+    <Box width="100%" padding="space60">
+      <FormLabel htmlFor="sort_do">Caller ID</FormLabel>
+      <Select id="sort_do" name="sort_do" onChange="">
+        <Option value="null">Choose a Caller ID</Option>
+        <Option value="+14155552287">+1(415)555-CATS</Option>
+        <Option value="+14155553647">+1(415)555-DOGS</Option>
+      </Select>
+    </Box>
+  </Do>
+  <Dont
+    center
+    title="Don't"
+    body="Don’t restrict users to options that unnecessarily reinforce biases and discrimination, such as gender binaries. Consider whether you need this information at all. If so, pair a select with an input or allow users to opt out."
+  >
+    <Box width="100%" padding="space60">
+      <FormLabel htmlFor="sort_do">Gender</FormLabel>
+      <Select id="sort_do" name="sort_do" onChange="">
+        <Option value="male">Male</Option>
+        <Option value="female">Female</Option>
+      </Select>
+    </Box>
+  </Dont>
+</DoDont>
+
+<DoDont>
+  <Do
+    center
+    title="Do"
+    body="Give users time to confirm their selection. Consider users who might select a wrong option accidentally."
+  />
+  <Dont
+    center
+    title="Don't"
+    body="Avoid triggering an action when a user selects an option. For example, don’t open a new page immediately after a user makes a selection. If they made the wrong selection, going back to the original page to make the right selection could be frustrating. Instead, allow users to make the selection, then confirm."
+  />
+</DoDont>
+
+<DoDont>
+  <Do
+    center
+    title="Do"
+    body="Keep option text concise and simple. If choosing options require more explanation, consider using a checkbox instead and add help text for each option, or give more explanation through help text."
+  >
+      <Box width="100%" padding="space60">
+      <FormLabel htmlFor="sort_do">HTTP method</FormLabel>
+      <Select id="sort_do" name="sort_do" onChange="">
+        <Option value="get">GET</Option>
+        <Option value="post">POST</Option>
+        <Option value="put">PUT</Option>
+      </Select>
+    </Box>
+  </Do>
+  <Dont
+    center
+    title="Don't"
+    body="Avoid long option text that will truncate if it’s longer than the width of the container."
+  >
+      <Box width="100%" padding="space60">
+      <FormLabel htmlFor="sort_do">Voice URL</FormLabel>
+      <Select id="sort_do" name="sort_do" onChange="">
```

Comment:
As a reminder, there is a bug here where the chevron overlaps the text. Please add a JIRA ticket for this